src/scripts/site.js

```diff
@@ -246,11 +246,24 @@ async function onAppReady() {
             }
         };
 
-        Events.on(ServerConnections, 'localusersignedin', handleStyleChange);
-        Events.on(ServerConnections, 'localusersignedout', handleStyleChange);
+        const handleLanguageChange = async () => {
```

Comment:
No need to be `async`.
```suggestion
        const handleLanguageChange = () => {
```